src/iso.js

```diff
@@ -35,17 +39,18 @@ ${data}
     )
   }
 
-  static render(html, state = {}, meta = {}) {
-    return new Iso().add(html, state, meta).render()
+  static render(html, state = {}, meta = {}, config = { markupClassName: '___iso-html___', markupElement: 'div', dataClassName: '___iso-state___', dataElement: 'div'}) {
+    return new Iso(config).add(html, state, meta).render()
   }
 
-  static bootstrap(onNode) {
+  static bootstrap(onNode, markupClassName = '___iso-html___', dataClassName = '___iso-state___') {
```

Comment:
Should bootstrap be static? I wasn't able to access the current instance scope here. I originally had it pulling from the config. 

See: https://github.com/michaelBenin/iso/commit/621e7aa6b2b57ae80b020d8f7b8b1d34afe36ae5#diff-4e0c48295f6c143af2dfba2e4bbca24cL76